training_scripts/gsgnn_np/ml_nc_utext_glem.yml

```diff
@@ -0,0 +1,56 @@
+---
+version: 1.0
+lm_model:
+  node_lm_models:
+    -
+      lm_type: bert
+      model_name: "bert-base-uncased"
+      gradient_checkpoint: false
+      node_types:
+        - movie
+        - user
+gsf:
+  basic:
+    backend: gloo
+    ip_config: ip_list.txt
+    part_config: movielen_100k_utext_train_val_1p_4t/movie-lens-100k.json
+    verbose: false
+    save_perf_results_path: null
+  lm:
+    lm_train_nodes: 10
+    lm_infer_batch_size: 32
+    freeze_lm_encoder_epochs: 0
+    glem:
+      em_order_gnn_first: true
+      inference_using_gnn: true
```

Comment:
is this argument necessary?

Reply:
Yes. I didn't implement `em_order_gnn_first`. Will add in the next commit.